packages/reporting-server/README.md

```diff
@@ -88,8 +88,148 @@ postgres://<user>:<password>@<host>/<database>
+
+The installation procedure created a user account called postgres that is associated with the default Postgres role. There are a few ways to utilize this account to access Postgres. One way is to switch over to the postgres account on your server by typing
+
+``` bash
+sudo -i -u postgres
+```
+
+Then you can access the Postgres prompt by typing:
+
+``` bash
+psql
+```
+
+This will log you into the PostgreSQL prompt, and from here you are free to interact with the database management system right away.
+
+We can create new user with the following command
+
+``` bash
+create user <youruser> with encrypted password <yourpassword>;
+
+# E.G: create user myuser with encrypted password 'mypassword';
+```
+
+After the user is created we need to assign the db roles to the user:
+
+``` bash
+grant all privileges on database <yourdatabase> to <youruser>;
+
+# E.G: grant all privileges on database reporting to myuser;
+```
+
+### Set/change user PostgreSQL password
+
+``` bash
+sudo -i -u postgres
+```
+
+Then you can access the Postgres prompt by typing:
+
+``` bash
+psql
+```
+
+To change your password you run the following command:
+
+``` bash
+\password <user>
+# E.g: \password postgres
+```
+
+To exit out of the PostgreSQL prompt, run the following:
+
+``` bash
+\q
+```
+
+
+
+### PostgresSQL URL
+
+Once the above steps are finished you can build your bd_url:
+
+``` 
+postgres://<user>:<password>@<host>/<database>
+```
+
+For more information you can check [this tutorial](https://www.digitalocean.com/community/tutorials/how-to-install-postgresql-on-ubuntu-20-04-quickstart)
+
+
+## Run the application
+
+Now that you have already elected, configured and created a database and set you configurations. You need to run the migrations to create the database tables. We can proceed to apply migrations to create the database tables by running
+
+```bash
+aerich upgrade
+```
+
+After running this command your database should have all the tables defined in the code. Now you can run the application
+
+```bash
+reporting_server
+```
+
+If you want to run the migrations and start the server you could also run:
+
+```bash
+npm run start:dev
+```
+
 # Developers
 
+## Migration
+We are using [aerich](https://github.com/tortoise/aerich) as our database migration tool. That means that the changes to made to the model will not be reflected in the database automatically. You must run the aerich migration command to create a new migration with the changes:
+
+``` bash
+npm run migrate
+```
+
+If there are some changes in the Tortoise models, the previous command will create a migration (sql) file that has your changes. To apply these changes to the database you need to run
+
+``` bash
+npm run apply:migrations 
+```
+
+![aerich](https://user-images.githubusercontent.com/11761240/122826198-d97f2e80-d2b0-11eb-813f-384f4ae61d6b.png)
+
```

Comment:
How do you run migrations if you installed `reporting-server` from "binary" (`pip3 install reporting-server`)?

Reply:
We cannot do that. Because we need to have the sql file generated and aerich.ini file to apply the migrations. 

If we copy the migrations folder and if we install the dependencies from binary we can run it.

Reply:
I guess the primary way to run this is in a docker container, so it is not designed to be usable with the "old school pip3 install"?

@codebot Is this a limitation a concern? Should we make the effort to ensure that it still works as a "traditional" software?

Reply:
Well, I think I depend on this answer to continue working on this feature. I'll switch to my other task in the meanwhile.